docs/topic-guides/data/data.rst

```diff
@@ -12,6 +12,12 @@ large files on the server and allows you to work with your repository without
 actually having a local copy of the data. You can pull the data from the server
 as it becomes needed, saving you time and resources.
 
+However, git LFS is not efficient when working with large amount of data.
+Hence, git LFS should be avoided, especially for big files, since such system usually
```

Comment:
This language feels too strongly against using `git lfs`. Also, which problems? This could simply mention that self-provisioned external storage can provide a more suitable solution in some cases.

Reply:
hmm idk, we do want to discourage people from using LFS for a lot of data though...

Reply:
also `such system` is a weird formulation instead of just `it` or `this`

Reply:
I tried to avoid controversy related to using the word "problems".